src/domain/service/auth.ts

```diff
@@ -1,9 +1,19 @@
 import jwt from 'jsonwebtoken';
 
 /**
- * Jwt service
+ * Payload interface
  */
-export default class JwtService {
+export interface AuthPayload {
+  /**
+   * User id
+   */
+  id: number;
```

Comment:
it is just unpacked JWT. When you will check user session in a Database?

Reply:
When we checks access, we don't not check session. We check user session, when he requests access by refresh, i think, we will use unpacking data for it, for example, session id